### DISCUSSION TITLE
Add More options for in-game shop

### DISCUSSION THREAD

**Discussion by Spencer021** [gameserverapp/Platform#576] [Suggestions]
Add a section that says "Limit to ___ number of purchases, ex: "Limit to 10 purchases" for shop packs
![image](https://github.com/gameserverapp/Platform/assets/58157745/efad524f-ca8e-4828-852a-0cbe05e31c5f)

**Comment by mvaessen** [comment]
This is already the case for in-game shop items, just like the web shop.
